src/world/World.php

```diff
@@ -2056,6 +2056,8 @@ public function useBreakOn(Vector3 $vector, Item &$item = null, ?Player $player
 		$drops = [];
 		if($player === null || $player->hasFiniteResources()){
 			$drops = array_merge(...array_map(fn(Block $block) => $block->getDrops($item), $affectedBlocks));
+		}elseif($player->isCreative()){
```

Comment:
This `$player->isCreative()` check is unnecessary, above is already checking `$player->hasFiniteResources()`, which in turn checks if the player is not on creative